### PR TITLE
improve design and fix translations

### PR DESCRIPTION
**Pull request by brunoOnTheWorld**
- fix job fonts
- fix missing translations
- fix the activated link on safari
- refactory modal
- include job button on the dashboard

### CODE REVIEW

File: src/printer/components/dataFormat.js [hunk 1]
```diff
@@ -54,16 +54,16 @@ function formatEstimatedTime(time) {
...
       end.getDate() == tomorrow.getDate() &&
       end.getMonth() == tomorrow.getMonth()
     ) {
-      plus_days = "tomorrow at ";
+      plus_days = translate("prop.tmw-at") + " ";
     } else {
       let options = { month: "numeric", day: "numeric" };
       const final_date = end.toLocaleString(window.navigator.language, options);
-      plus_days = `${final_date} at `;
+      plus_days = `${final_date} ${translate("prop.at")} `;
```
Comment (Lowentwickler) [inline src/printer/components/dataFormat.js:66]: Have you tested this? If there will be some issue with missing translation, just assign a variable first
```
const at = translate("prop.at");
plus_days = `${final_date} ${at} `;
```

File: src/printer/components/dataFormat.js [hunk 2]
```diff
@@ -54,16 +54,16 @@ function formatEstimatedTime(time) {
...
     let plus_days = "";
     if (end.getDate() == now.getDate() && end.getMonth() == now.getMonth()) {
-      plus_days = "today at ";
+      plus_days = translate("prop.today-at") + " ";
     } else if (
       end.getDate() == tomorrow.getDate() &&
       end.getMonth() == tomorrow.getMonth()
     ) {
-      plus_days = "tomorrow at ";
+      plus_days = translate("prop.tmw-at") + " ";
```
Comment (Lowentwickler) [inline src/printer/components/dataFormat.js:62]:
```
      plus_days = `${translate("prop.tmw-at")} `;
```

**Review by Lowentwickler — APPROVED** [review]

File: src/printer/components/dataFormat.js (hunk 1 above)
Comment (brunoOnTheWorld) [reply src/printer/components/dataFormat.js:66]: indeed this worked on my computer

Comment (brunoOnTheWorld) [reply src/printer/components/dataFormat.js:66]: I'll change for your suggestion
Thank you

**Pull request merged**